pom.xml

```diff
@@ -41,7 +41,12 @@
 		<repository>
 			<id>jenkins-releases</id>
 			<name>Jenkins Releases</name>
-			<url>http://repo.jenkins-ci.org/releases</url>
+			<url>https://repo.jenkins-ci.org/releases</url>
+		</repository>
+		<repository>
+			<id>repo.jenkins-ci.org</id>
+			<name>Jenkins Repo</name>
+			<url>https://repo.jenkins-ci.org/public</url>
```

Comment:
Is it really necessary to include this repo? As it wasn't included before I take it as it is not requierd as no deps have been added/changed.
Also, that "public" repo is a repo that merges several repositories (including central it seems). It might possibly be simple to use that in your settings.xml, but from a Maven perspective it's not ideal to include it in a pom.xml for an artifact that will be used as a dependency. Keeping it in the examples could be ok though.

Reply:
I don't recall since I opened this nearly a year ago. But I believe I added it to get this project to build on master.

Reply:
This is necessary for Maven 3.8.x since inherited dependencies will try to be fetched from `http://repo.jenkins-ci.org/public` as specified in `https://repo.jenkins-ci.org/artifactory/releases/settings.xml`, this overrides that and allows master to build

Reply:
I just refreshed myself on this. It is absolutely required because the current master branch is broken, hence the title to my pr "Fix master". Marking this as resolved.